### PR TITLE
Add list task command

### PR DESCRIPTION
**Pull request by xinweichong**

### CODE REVIEW

File: src/test/java/seedu/address/logic/commands/ListTaskCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    @BeforeEach
+    public void setUp() {
+        model = new ModelManager(getTypicalAddressBookWithTasks(), new UserPrefs());
+        expectedModel = new ModelManager(model.getAddressBook(), new UserPrefs());
+    }
+
+    @Test
+    public void execute_listIsNotFiltered_showsSameList() {
+        assertCommandSuccess(new ListTaskCommand(), model, ListTaskCommand.MESSAGE_SUCCESS, expectedModel);
+    }
```
Comment (gavinsoh) [inline src/test/java/seedu/address/logic/commands/ListTaskCommandTest.java:32]: I like how you check that the list command works as expected even when no tasks are filtered out.

File: src/test/java/seedu/address/logic/commands/ListTaskCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+public class ListTaskCommandTest {
+
+    private Model model;
+    private Model expectedModel;
+
+    @BeforeEach
+    public void setUp() {
+        model = new ModelManager(getTypicalAddressBookWithTasks(), new UserPrefs());
+        expectedModel = new ModelManager(model.getAddressBook(), new UserPrefs());
+    }
```
Comment (gavinsoh) [inline src/test/java/seedu/address/logic/commands/ListTaskCommandTest.java:27]: I like how you used @BeforeEach to initialise models for each test as it helps in ensuring test isolation.

File: src/test/java/seedu/address/logic/commands/ListTaskCommandTest.java [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    public void execute_listIsNotFiltered_showsSameList() {
+        assertCommandSuccess(new ListTaskCommand(), model, ListTaskCommand.MESSAGE_SUCCESS, expectedModel);
+    }
+
+    @Test
+    public void execute_listIsFiltered_showsEverything() {
+        showTaskAtIndex(model, INDEX_FIRST_TASK);
+        assertCommandSuccess(new ListTaskCommand(), model, ListTaskCommand.MESSAGE_SUCCESS, expectedModel);
+    }
+}
```
Comment (benjiBase) [inline src/test/java/seedu/address/logic/commands/ListTaskCommandTest.java:39]: nice test cases with coverage for different situations

File: src/main/java/seedu/address/logic/commands/ListTaskCommand.java [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_TASKS;
+
+import seedu.address.model.Model;
+
+/**
+ * Lists all persons in the address book to the user.
+ */
+public class ListTaskCommand extends Command {
+
+    public static final String COMMAND_WORD = "listtask";
```
Comment (benjiBase) [inline src/main/java/seedu/address/logic/commands/ListTaskCommand.java:13]: nice choice of command word that suits the purpose of what users want to do

**Review by gavinsoh — APPROVED** [review]

**Pull request merged**